### DISCUSSION TITLE
Welcome to segment-geospatial Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [opengeos/segment-geospatial#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shamukshi** [comment]
For "solar panel counting from UAV image using SAM"....
which approach is better ? 1. object detection and segmentation or 2. segmenatation and classification.
